autoparams/src/main/java/org/javaunit/autoparams/generator/IntegerGenerator.java

```diff
@@ -30,4 +32,21 @@ private int getOrigin(ObjectQuery query) {
         return Integer.MIN_VALUE;
     }
 
+    private int getMaximum(ObjectQuery query) {
+        if (query instanceof ArgumentQuery) {
+            ArgumentQuery argumentQuery = (ArgumentQuery) query;
+            Max annotation = argumentQuery.getParameter().getAnnotation(Max.class);
+            if (annotation != null) {
+                return (int) Math.max(
```

Comment:
이렇게 하면 코드가 짧아지고 모든 테스트가 통과할 것 같습니다.

```suggestion
                return (int) Math.min(Integer.MAX_VALUE, annotation.value());
```

Reply:
@gyuwon 변경사항 반영했습니다. squash 해서 최종적으로 다시 올릴까요?

Reply:
@Rebwon 직접 판단해주세요. Squash 하실 거면 기다리겠습니다. :)

Reply:
@gyuwon squash 했습니다. 기여하는 동안 테스트 작성하신 부분과 변경사항에 대해 좋은 학습이 되었습니다. 리뷰 감사합니다 :)

Reply:
@Rebwon 수고 많으셨습니다 :)